Add the usage of install.sh in README.md (#247)
pingcap/chaos-mesh: doc/get_started_on_kind.md

# Get started on kind

This document describes how to deploy Chaos Mesh in Kubernetes on your laptop (Linux or macOS) using kind.

## Prerequisites

Before deployment, make sure [Docker](https://docs.docker.com/install/) is installed and running on your local machine.

> ## Step 1: Get the Helm files

Reviewer: ditto

```bash
git clone https://github.com/pingcap/chaos-mesh.git
cd chaos-mesh/
```

## Step 2: Install Chaos Mesh

```bash
./install.sh --local kind
```

`install.sh` is an automation shell script that helps you install dependencies such as `kubelet`, `Helm`, `kind`, and `kubernetes`, and `Chaos Mesh` itself.

After executing the above command, you should be able to see the prompt that Chaos Mesh is installed successfully. 
Otherwise, please check the current environment according to the prompt message or send us an [issue](https://github.com/pingcap/chaos-mesh/issues) for help. 
In addition, You also can use [Helm](https://helm.sh/) to [install Chaos Mesh manually](deploy.md).

## Next steps

> Refer to [Run Chaos Mesh](run_chaos_mesh.md)

Reviewer 2: ```suggestion
Refer to [Run Chaos Mesh](run_chaos_mesh.md).
```

# Get started on kind

This document describes how to deploy Chaos Mesh in Kubernetes on your laptop (Linux or macOS) using kind.

## Prerequisites

Before deploying Chaos Mesh, make sure the following items have been installed. 

> - [Docker](https://docs.docker.com/install/) (required when running in [kind](https://kind.sigs.k8s.io/))

Reviewer 2: If there is only 1 item, you don't have to use a list.  Just combine the sentence into one paragraph.

## Step 1: Get the Helm files

```bash
git clone https://github.com/pingcap/chaos-mesh.git
cd chaos-mesh/
```

## Step 2: Install Chaos Mesh

```bash
./install.sh --local kind
```

> `install.sh` will help you to install `kubelet`, `Helm`, `kind`, `kubernetes` and `Chaos Mesh`. 

Reviewer 2: ```suggestion
`install.sh` is an automation shell script that helps you install `kubelet`, `Helm`, `kind`, `kubernetes` and `Chaos Mesh`. 
```

Reviewer 2: use the present tense as much as possible.

After executing the above command, if the message that Chaos Mesh is installed 
successfully is output normally, then you can continue next steps to test your application and enjoy Chaos Mesh. 
Otherwise, please check the current environment according to the prompt message of the script 
or send us an [issue](https://github.com/pingcap/chaos-mesh/issues/new/choose) for help. 
In addition, You also can use [Helm](https://helm.sh/) to [install Chaos Mesh manually](deploy.md).

> 

Reviewer 2: After executing the above command, if you should be able to see the prompt that Chaos Mesh is installed successfully. Otherwise, please check the current environment according to the prompt message or send us an [issue](https://github.com/pingcap/chaos-mesh/issues/new/choose) for help.

## Step 3: Run Chaos Mesh

> Refer to the Steps in [Run Chaos Mesh](run_chaos_mesh.md)

Reviewer 2: ```suggestion
Refer to [Run Chaos Mesh](run_chaos_mesh.md).
```
